### PR TITLE
Update LICENSE.txt - minor correction for consistency

### PR DESCRIPTION
**Pull request by manez**
# What does this Pull Request do?

Line 655 was different from other licences. This update brings it in line with the rest and with the example from [gnu.org](https://www.gnu.org/licenses/gpl-3.0.en.html)

# How should this be tested?

Pull up the licence from a few other modules, compare the new line to see that it matches what's used elsewhere. Compare to the wording from gnu.org.

# Interested parties
@DiegoPino or maintainer @jonathangreen

### CODE REVIEW

**Review by DiegoPino — APPROVED** [review]

File: LICENSE.txt [hunk 1]
```diff
@@ -652,7 +652,7 @@ Also add information on how to contact you by electronic and paper mail.
   If the program does terminal interaction, make it output a short
 notice like this when it starts in an interactive mode:
 
-    {{project}}  Copyright (C) {{year}}  {{fullname}}
+    Copyright (C) <year> <name of author>
```
Comment (DiegoPino) [inline LICENSE.txt:655]: Melissa. I know its just a tiny thing, but we should be consistent with `{}` or `<>` use across the doc.  Not sure if better to decide which version of the originals https://www.gnu.org/licenses/gpl-3.0.txt or https://choosealicense.com/licenses/gpl-3.0/ we want to use and replace the whole thing. What do you think?

Comment (manez) [reply LICENSE.txt:655]: @DiegoPino I think https://www.gnu.org/licenses/gpl-3.0.txt is the standard we're using, not the other

Comment (DiegoPino) [reply LICENSE.txt:655]: Cool. Then we need to change https://github.com/Islandora/islandora_bagit/pull/43/files#diff-61e0bdf7e1b43c5c93d9488b22e04170R667 and other {} here. Thanks!

Comment (manez) [reply LICENSE.txt:655]: I'll just close this pr and redo the whole thing based off the standard to make sure we get it all.

**Pull request closed without merging**